### PR TITLE
Update hosting-on-firebase.md

### PR DESCRIPTION
**Pull request by bggo**
Firebase now does support native integration with Github Actions for CICD. Including steps on this documentation.

### CODE REVIEW

File: content/en/hosting-and-deployment/hosting-on-firebase.md [hunk 1]
```diff
@@ -52,15 +52,47 @@ From here:
 4. Accept the default for the publish directory, which is `public`
 5. Choose "No" in the question if you are deploying a single-page app
 
-## Deploy
+## Using Firebase & Github CI/CD
```
Comment (coliff) [inline content/en/hosting-and-deployment/hosting-on-firebase.md:55]: Github should be GitHub

File: content/en/hosting-and-deployment/hosting-on-firebase.md [hunk 2]
```diff
@@ -52,15 +52,47 @@ From here:
...
+
+In new versions of firebase, some other questions apply:
```
Comment (coliff) [inline content/en/hosting-and-deployment/hosting-on-firebase.md:57]: Capitalize Firebase

File: content/en/hosting-and-deployment/hosting-on-firebase.md [hunk 3]
```diff
@@ -52,15 +52,47 @@ From here:
...
+
+After that firebase has been set in your project with CI/CD. After that run:
+
+```
+hugo && firebase deploy
+```
+
+With this you will have the app initialized manualy. After that you can manage and fix your github workflow from: https://github.com/your-account/yout-repo/actions
+
+Dont forget to update your static pages before push!
```
Comment (coliff) [inline content/en/hosting-and-deployment/hosting-on-firebase.md:84]: Don't

File: content/en/hosting-and-deployment/hosting-on-firebase.md [hunk 4]
```diff
@@ -52,15 +52,47 @@ From here:
...
+In new versions of firebase, some other questions apply:
+
+6. Set up automatic builds and deploys with GitHub? 
+
+Here you will be redirected to login in your github account to get permissions. Confirm.
+
+7. For which GitHub repository would you like to set up a GitHub workflow? (format: user/repository) 
+
+Include the repository you will use in the format above (Account/Repo)
+Firebase script with retrive credentials, create a service account you can later manage in yout github settings.
```
Comment (coliff) [inline content/en/hosting-and-deployment/hosting-on-firebase.md:66]: should be 'manage in your GitHub'

**Review by coliff — CHANGES_REQUESTED** [review]
Thanks for the PR. Please check the spelling and fix any capitalization issues. (e.g. Firebase and GitHub should be capitalized like that)

**Comment by bep** [comment]
@bggo is this still on your ... agenda (re @coliff 's remarks above)?

**Pull request merged**